### PR TITLE
Snowflake rewards

### PR DESCRIPTION
**Pull request by tasinco**

### CODE REVIEW

File: indexer/persistor_tasks.go [hunk 1]
```diff
@@ -292,28 +299,84 @@ func (t *systemEventPersistorTask) Run(ctx context.Context, p pipeline.Payload)
...
+	rewardStore *RewardStore
 }
 
 func (t *RewardEraSeqPersistorTask) GetName() string {
 	return RewardEraSeqPersistorTaskName
 }
 
+type rewardJSONView struct {
+	Height                int64            `json:"height"`
+	Era                   int64            `json:"Era"`
```
Comment (christophermanning) [inline indexer/persistor_tasks.go:320]:
```suggestion
	Era                   int64            `json:"era"`
```

File: indexer/persistor_tasks.go [hunk 2]
```diff
@@ -292,28 +299,84 @@ func (t *systemEventPersistorTask) Run(ctx context.Context, p pipeline.Payload)
...
+	StartHeight           int64            `json:"start_height"`
+	EndHeight             int64            `json:"end_height"`
+	Time                  types.Time       `json:"time"`
+	StashAccount          string           `json:"stash_account"`
+	ValidatorStashAccount string           `json:"validator_stash_account"`
+	Amount                string           `json:"amount"`
+	Kind                  model.RewardKind `json:"kind"`
+	Chunk                 uint64           `json:"chunk"`
+	MaxChunk              uint64           `json:"maxChunk"`
```
Comment (christophermanning) [inline indexer/persistor_tasks.go:329]:
```suggestion
	MaxChunk              uint64           `json:"max_chunk"`
```

File: helm/pipeline-polkadot/values.yaml [hunk 3]
```diff
@@ -151,6 +161,10 @@ deployments:
               secretsManagerKeyRef:
                 key: production/indexers/pipeline-polkadot
                 property: datastore_url
+            REWARDS_HTTP_STORE_ADDRESS:
+              secretsManagerKeyRef:
+                key: production/indexers/pipeline-westend
```
Comment (christophermanning) [inline helm/pipeline-polkadot/values.yaml:166]: should this be the pipeline-polkadot key?

File: helm/pipeline-polkadot/values.yaml [hunk 4]
```diff
@@ -81,6 +85,10 @@ deployments:
...
```
Comment (christophermanning) [inline helm/pipeline-polkadot/values.yaml:90]: should this be the pipeline-polkadot key?

File: helm/pipeline-polkadot/values.yaml [hunk 5]
```diff
@@ -14,6 +14,8 @@ jobs:
         pipeline-polkadot-mainnet-migration:
           env:
             APP_ENV: production
+            NETWORK: "polkadot"
+            CHAIN_ID: "migration"
```
Comment (christophermanning) [inline helm/pipeline-polkadot/values.yaml:18]: why is this chain_id migration?

Comment (tasinco) [reply helm/pipeline-polkadot/values.yaml:18]: I just have a check for the chain_id not to be nil..  the migration doesn't care what the value is.

**Review by christophermanning — APPROVED** [review]

**Pull request merged**